### DISCUSSION TITLE
SD.Next Gui do not show the Extra Tab for the extension

### DISCUSSION THREAD

**Discussion by Andy1977** [vladmandic/sdnext#2382] [General]
Hallo all,

I want to use the superb background removal extension.
The extension is loaded, which I can see on the terminal.
But I do not know how I can use this extension - or extensions generally.

I see the dropdown listbox - with the script dropdown listbox above,
but the listbox is empty.

The extensions is also activated.

Greetings

Andy

**Comment by Tillerz** [comment]
Whch extension exactly? stable-diffusion-webui-rembg ?

**Reply by Andy1977** [reply]
Yes, I want to use the "stable-diffusion-webui-rembg".

**Reply by vladmandic** [reply]
Not every extension has a tab!!!
That extension works just fine. 
How do you use it? Select it as a task in processing.
